kernel/fde/cmd_helper.go

```diff
@@ -49,7 +49,7 @@ var fdeInitramfsHelperPollWaitParanoiaFactor = 2
 // overridden in tests
 var fdeInitramfsHelperCommandExtra []string
 
-func runFDEinitramfsHelper(name string, stdin []byte) (output []byte, err error) {
+func RunFDEinitramfsHelper(name string, stdin []byte) (output []byte, err error) {
```

Comment:
@kubiko @valentindavid my wondering is whether we should instead export a function Setup or SetupFromInitramfs that has already the RunSetupHookFunc signature, this would be more consistent/symmetrical with Reveal vs fed-reveal-key here